api/error.go

```diff
@@ -17,8 +17,6 @@ limitations under the License.
 package api
 
 import (
-	"fmt"
-
 	"github.com/pkg/errors"
```

Comment:
Is this import intentional or by mistake? I think we should use go built-in `errors`.

Reply:
The built-in `errors` doesn't have `errors.Wrap`.  IMO `fmt.Errorf("foo: %w", err)` would be more idiomatic but I think that's out of the scope of this PR.